src/configManager/controllers/configController.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+let configModel = require('../models/configHandler');
+
+let logger = require('../../common/logger');
+
+module.exports.getConfig = function (req, res, next) {
+    logger.info('inside getConfig()');
+    configModel.getConfig().then((response) => {
+        return res.status(200).json(response);
+    }).catch((err) => {
+        return next(err, res);
+    });
+};
+
+module.exports.updateConfig = function (req, res, next) {
+    logger.info('inside updateConfig()');
+    let body = req.body;
+    configModel.updateConfig(body).then(() => {
+        return res.status(200).json(body);
+    }).catch((err) => {
```

Comment:
its wrong to get a body and return it like this. 
if we get body with extra fields, you will return it  with the extra fields. maybe you should return the database result(mapped to what you want)?

Reply:
We didn't want to return all config from DB, so we wanted to return the body.
if we use Swager config you cant send not valid data in the body, no?